src/benchq/problem_ingestion/molecule_instance_generation.py

```diff
@@ -180,6 +180,9 @@ def get_occupied_and_active_indicies_with_FNO(
         molecular_data.canonical_orbitals = natural_orbital_coefficients
         mean_field_object.mo_coeff = molecular_data.canonical_orbitals
 
+        if mean_field_object is None:
+            raise ValueError("WTF??? Why are you zero boi?")
```

Comment:
Did you encounter cases where `mean_field_object` was `None`? If so we should probably investigate and address it in a separate PR.

Reply:
This is the horrible bug that I've been dealing with. I didn't think this check was on the main PR. Oops... Maybe I should make another PR with minimal changes and try again.